stable/database/values.yaml

```diff
@@ -351,6 +351,10 @@ database:
     #Some clusters require longer readiness probe timeouts
     readinessTimeoutSeconds: 5
   
+  #These annotations will pass throgh to the pod as supplied, useful for integrating 3rd party products such as Vault.
+  annotations: {}
```

Comment:
It seems not consistent with the variable used in the template.

Reply:
Thanks, fixed